### DISCUSSION TITLE
plugin not fully functioning, any dependencies/known issues/etc?

### DISCUSSION THREAD

**Discussion by lobosaid-g** [chhoumann/quickadd#83] [Q&A]
This is a really interesting plugin, however, for me, it only works with one combination, i.e., when I have 'Capture to active file' and 'Write to the bottom of the file' enabled. If I enable any other option, the capture doesn't work (I can input data but it just doesn't do anything). So, I was wondering, is there any requirement or something else that might be interfering with the plugin?

Thanks!

**Comment by chhoumann** [comment]
Hey @lobosaid-g!

That's very odd. Can you provide me with some steps to reproduce?
Are there any errors in the developer console? If you use CTRL + SHIFT + i (Windows), it'll show it.

**Reply by lobosaid-g** [reply]
Yeah, that's actually weird... the error I'm getting is:
`QuickAdd: (LOG) TypeError: Cannot read property 'parse_template' of undefined`
So, the issue is between this plugin and Templater plugin, if I disable the Templater, this plugin works fine

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

Closing the loop: the `parse_template` error you found was QuickAdd reaching into Templater's internals without guarding against a version mismatch, which is why disabling Templater made captures work again. That integration has been completely rewritten since - QuickAdd now checks that Templater's API is actually available before calling it, and falls back gracefully if it isn't, so this combination can't break captures anymore.

Closing as outdated since this is 2021-era behavior. If anything similar shows up on a current QuickAdd, please open a fresh issue with the console output and I'll take a look.
